src/CommitmentsV2/SFA.DAS.CommitmentsV2/Data/AccountLegalEntityConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SFA.DAS.CommitmentsV2.Models;
+
+namespace SFA.DAS.CommitmentsV2.Data
+{
+    public class AccountLegalEntityConfiguration : IEntityTypeConfiguration<AccountLegalEntity>
+    {
+        public void Configure(EntityTypeBuilder<AccountLegalEntity> builder)
+        {
+            builder.Property(ale => ale.Id).ValueGeneratedNever();
+            builder.Property(ale => ale.PublicHashedId).IsRequired().HasColumnType("char(6)");
```

Comment:
nchar(6)?

Reply:
no they can only be char in the DB. The hashing values are limited to A-Z1-9

Reply:
changed